ui/components/src/ThresholdsEditor/ThresholdsEditor.tsx

```diff
@@ -116,7 +119,7 @@ export function ThresholdsEditor({ thresholds, onChange, hideDefault, disablePer
 
   const addThresholdInput = (): void => {
     focusRef.current = true;
-    if (thresholds === undefined) {
+    if (thresholds?.steps === undefined) {
       onChange({
         steps: [{ value: DEFAULT_STEP }],
       });
```

Comment:
Could this overwrite other values on `thresholds`? Like if `thresholds = { default_color: blah-blah }`, would this overwrite / lose the `default_color` field?

But I'm not sure if that would happen

Reply:
good catch!!